### PR TITLE
added badges and buttons component

### PR DESCRIPTION
**Pull request by rajaranjanmr**

### CODE REVIEW

File: badges/badges1.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+:root{
+    --primaryBadgeCol:#876445;
+    --secondaryBadgeCol:#D96098;
+    --thirdBadgeCol:#24A19C;
+    --fourthBadgeCol:#325288;
+}
+
+.badges{
+    color:rgb(202, 186, 186);
```
Comment (parvez-rashid) [inline badges/badges1.css]: Use variable for color here. Also use consistent units in colors: either all RGB or all hex values (hex is easier).

File: badges/badgesAvtaar.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
+.avtaar-badges{
+    display:flex;
+    gap:1rem;
+    flex-direction: row;
```
Comment (parvez-rashid) [inline badges/badgesAvtaar.css]: You don't have to explicitly mention `flex-direction: row` as by default `display: flex` takes row. Just to make code more concise.

File: badges/badgesAvtaar.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    width: 15%;
+    height: 15%;
+    position: relative;
+}
+/* .sm-badges{
+    width: 20%;
+    height: 20%;
+    position: relative;
+
+} */
```
Comment (parvez-rashid) [inline badges/badgesAvtaar.css]: Remove the comment; same class has been declared above as well so I'm assuming the comment isn't needed anymore.

File: badges/badges1.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="badges1.css">
+    <link rel="stylesheet" href="/Users/raranjan/Documents/component library/component-librarys/clhome.css">
```
Comment (parvez-rashid) [inline badges/badges1.html]: Have a look at the path of the second css file. You can move the file to the project's folder, like you did the first css file.

File: badges/badgesIcon.css [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+.badgeIcon{
+    font-family: sans-serif;
+    color:black;
+    position: relative;
+    
+}
+.badge-icon1{
+    font-size: 3rem;
+    color:var(--badgeIconGreyCol);
+}
```
Comment (parvez-rashid) [inline badges/badgesIcon.css]: Here class names are similar. May get confusing later. Also use similar naming style: either use camelCase or kebab-case. Not both.

**Review by parvez-rashid — APPROVED** [review]
Have left comments below, please have a look. All the best!

File: buttons/button.css [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+padding-top: 0.7em;
+padding-bottom: 0.5rem;
+color: white;
+background-color: #082032;
+text-align: center;
+cursor: pointer;
+font-size: 18px;
+border-radius: 10px;
+text-align: center;
+font-weight: 800;
```
Comment (sri-rishi) [inline buttons/button.css]: give tab spacing to look consistent .

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Screenshot (30)

 Buttons are getting top on one other, fix it, and also icons should have inlined  and should have same color as text

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Screenshot (32)

Give badge same height and width to make it round

**Pull request merged**